src/DocumentationBrowserViewExtension/DocumentationBrowserView.xaml.cs

```diff
@@ -170,22 +170,30 @@ async void InitializeAsync()
                     };
                 }
 
-                //Initialize the CoreWebView2 component otherwise we can't navigate to a web page
-                await documentationBrowser.EnsureCoreWebView2Async();
-           
-                this.documentationBrowser.CoreWebView2.WebMessageReceived += CoreWebView2OnWebMessageReceived;
-                comScriptingObject = new ScriptingObject(this.viewModel);
-                //register the interop object into the browser.
-                this.documentationBrowser.CoreWebView2.AddHostObjectToScript("bridge", comScriptingObject);
-
-                this.documentationBrowser.CoreWebView2.Settings.IsZoomControlEnabled = true;
-                this.documentationBrowser.CoreWebView2.Settings.AreDevToolsEnabled = true;
-
-                initState = AsyncMethodState.Done;
+                try
+                {
+                    //Initialize the CoreWebView2 component otherwise we can't navigate to a web page
+                    await documentationBrowser.Initialize(Log);
+     
+                    this.documentationBrowser.CoreWebView2.WebMessageReceived += CoreWebView2OnWebMessageReceived;
+                    comScriptingObject = new ScriptingObject(this.viewModel);
+                    //register the interop object into the browser.
+                    this.documentationBrowser.CoreWebView2.AddHostObjectToScript("bridge", comScriptingObject);
+
+                    this.documentationBrowser.CoreWebView2.Settings.IsZoomControlEnabled = true;
+                    this.documentationBrowser.CoreWebView2.Settings.AreDevToolsEnabled = true;
+
+                    initState = AsyncMethodState.Done;
+                }
+                catch(ObjectDisposedException ex)
```

Comment:
should we be catching other exceptions?

Reply:
ObjectDisposedException is the only exception type that we should expect to happen, and mostly in tests. It should not trigger crashes and Dynamo should be able to go on running.
For other types of exceptions, we should let Dynamo handle it (crash)